scalereg/scale_templates/reg6/reg_kiosk_clear.html

```diff
@@ -0,0 +1,42 @@
+<html>
```

Comment:
I'd just drop all the kiosk files for now, since the code that uses it hasn't been checked in yet.